src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/files/jupyterhub/02-spawner.py

```diff
@@ -63,6 +65,7 @@ def get_conda_store_environments(query_package: str = ""):
         "/usr/local/etc/jupyterhub/jupyterhub_config.py"
     )
     c.JAppsConfig.hub_host = "hub"
+    c.JAppsConfig.service_workers = 4
```

Comment:
@aktech What is going on with this line? It seems unrelated to the issue.

Reply:
Yep, this is unrelated. This is harnessing the new feature being able to configure workers for jhub-apps service.  I thought I would add all jhub apps related settings in this PR, but happy to create an new PR for this.